<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>0</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="WidgetSocketView" name="widgetSocketView" native="true"/>
      <widget class="WidgetTabPages" name="tabPages" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>新建</string>
    </property>
    <addaction name="actionAddTcpServer"/>
    <addaction name="actionAddTcpClient"/>
    <addaction name="actionAddUdpServer"/>
    <addaction name="actionAddUdpClient"/>
    <addaction name="actionAddUDPBroadcast"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionVersion"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionJSONFormat"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAddTcpServer">
   <property name="text">
    <string>TCP 服务器</string>
   </property>
   <property name="toolTip">
    <string>TCP 服务器</string>
   </property>
  </action>
  <action name="actionAddTcpClient">
   <property name="text">
    <string>TCP 客户端</string>
   </property>
   <property name="toolTip">
    <string>TCP 客户端</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>版本</string>
   </property>
   <property name="toolTip">
    <string>版本</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
  </action>
  <action name="actionAddUdpServer">
   <property name="text">
    <string>UDP 服务端</string>
   </property>
   <property name="toolTip">
    <string>新增 UDP 服务端</string>
   </property>
  </action>
  <action name="actionAddUdpClient">
   <property name="text">
    <string>UDP 客户端</string>
   </property>
   <property name="toolTip">
    <string>新增 UDP 客户端</string>
   </property>
  </action>
  <action name="actionAddUDPBroadcast">
   <property name="text">
    <string>UDP 　广播</string>
   </property>
   <property name="toolTip">
    <string>UDP 　广播</string>
   </property>
  </action>
  <action name="actionJSONFormat">
   <property name="text">
    <string>JSON 格式化</string>
   </property>
   <property name="toolTip">
    <string>JSON 格式化</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetTabPages</class>
   <extends>QWidget</extends>
   <header>Widget/widgettabpages.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetSocketView</class>
   <extends>QWidget</extends>
   <header location="global">Widget/widgetsocketview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
